src/classBased/components/TodoItem.js

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import styles from "./TodoItem.module.scss"
+
+class TodoItem extends React.Component {
+  handleEditing = () => {
+    this.setState({
+      editing: true,
+    })
+  }
+  state = {
+    editing: false,
+  }
+  handleUpdatedDone = event => {
+    if (event.key === "Enter") {
+      this.setState({ editing: false })
+    }
+  }
+  componentWillUnmount() {
+    console.log("Cleaning up...")
```

Comment:
Kindly remove the console.log as it is advisable to use only when  writing your code to test some scenarios